EIPS/eip-8101.md

```diff
@@ -0,0 +1,446 @@
+---
+eip: 8101
+title: Payload Chunking with Chunk Access Lists
+description: Semantic block chunking with separated state diffs for streaming validation and reduced latency
+author: Toni Wahrstätter (@nerolation), Milos Stankovic (@morph-dev), Jihoon Song (@jihoonsong), Bharath Vedartham (@bharath-123), Raúl Kripalani (@raulk)
+discussions-to: https://ethereum-magicians.org/t/eip-8101-payload-chunking/27085
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-12-01
+requires: 4844, 4895, 7732, 7825, 7928
+---
+
+## Abstract
+
+This EIP introduces Payload Chunking, a protocol upgrade that restructures Ethereum block propagation into self-contained execution chunks with separated Chunk Access Lists (CALs). Proposer propagates beacon and execution block headers, followed by CALs and Chunks. CALs propagate separately and contain state diffs that enable independent chunk execution. Chunks are executed and verified separately (as long as CALs `0..N` are available for chunk `N`). After all chunks execute, the CL validates and finalizes the block. This architecture transforms block validation from a monolithic operation into a streaming pipeline while preserving block atomicity. The concept of chunks is used only during block propagation.
```

Comment:
> The concept of chunks is used only during block propagation.

The concept of chunks is used during block construction, propagation and execution. Additionally, during syncing, now a node would request for chunks and CALs. It seems the concept of chunks replaces that of block.

A follow-up question would be does the EL need to construct a block from chunks for internal usage and storage.